python-sdk/pyproject.toml

```diff
@@ -82,7 +82,9 @@ all = [
     "snowflake-sqlalchemy>=1.2.0,<=1.2.4",
     "sqlalchemy-bigquery>=1.3.0",
     "s3fs",
-    "protobuf<=3.20", # Google bigquery client require protobuf <= 3.20.0. We can remove the limitation when this limitation is removed
+    # Google bigquery client require protobuf <= 3.20.0. We can remove the limitation when this limitation is removed.
+    # And additionally there is a security issue for versions >=3.20.0, <3.20.2, so restrict it to <3.20
+    "protobuf<3.20",
```

Comment:
If you want this change to land in Python SDK 1.2 -- please talk to @sunank200 / @phanikumv -- otherwise you will have to handle this on the SQL CLI end

Reply:
@tatiana said we can hold on to this at the moment.